site/content/installation/nginx-plus-jwt.md

```diff
@@ -0,0 +1,240 @@
+---
+title: "NGINX Plus Image and JWT Requirement"
+weight: 300
+toc: true
+docs: "DOCS-000"
+---
+
+## Overview
+
+NGINX Gateway Fabric with NGINX Plus requires a valid JSON Web Token (JWT) to download the container image from the F5 registry. In addition, starting with version 1.5.0, this JWT token is also required to run NGINX Plus.
+
+This requirement is part of F5’s broader licensing program and aligns with industry best practices. The JWT will streamline subscription renewals and usage reporting, helping you manage your NGINX Plus subscription more efficiently. The [telemetry](#telemetry) data we collect helps us improve our products and services to better meet your needs.
+
+The JWT is required for validating your subscription and reporting telemetry data. For environments connected to the internet, telemetry is automatically sent to F5’s licensing endpoint. In offline environments, telemetry is routed through [NGINX Instance Manager](https://docs.nginx.com/nginx-management-suite/nim/). Usage is reported every hour and on startup whenever NGINX is reloaded.
+
+## Setting up the JWT
+
+The JWT needs to be configured before deploying NGINX Gateway Fabric. We'll store the JWT in two Kubernetes Secrets. One will be used for downloading the NGINX Plus container image, and the other for running NGINX Plus.
+
+{{< include "installation/jwt-password-note.md" >}}
+
+### Download the JWT from MyF5
+
+{{<include "installation/nginx-plus/download-jwt.md" >}}
+
+### Docker Registry Secret
+
+{{<include "installation/nginx-plus/docker-registry-secret.md" >}}
+
+Provide the name of this Secret when installing NGINX Gateway Fabric:
+
+{{<tabs name="docker-secret-install">}}
+
+{{%tab name="Helm"%}}
+
+Specify the Secret name using the `nginxGateway.serviceAccount.imagePullSecret` or `nginxGateway.serviceAccount.imagePullSecrets` helm value.
```

Comment:
I know this line already exists, but isn't `serviceAccount` just under values in the chart? Like its not under `nginxGateway`. 

https://github.com/nginxinc/nginx-gateway-fabric/tree/main/charts/nginx-gateway-fabric#:~:text=serviceAccount.imagePullSecret

Reply:
Yes, good catch.